packages/oui-pagination/README.md

```diff
@@ -0,0 +1,124 @@
+# oui-pagination
+
+<component-status cx-design="partial" ux="prototype"></component-status>
+
+oui-pagination is a package which provide styles for the pagination component.
+
+## Installation
+
+```less
+@import 'oui-pagination/pagination';
+```
+
+## Usage
+
+### Button group
+
+```html:preview
+<div class="oui-pagination">
+  <div class="oui-pagination__progress">
+    <oui-dropdown arrow>
+      <button type="button"
+          class="oui-button oui-button_link"
+          oui-dropdown-trigger>
+          1 - 25</button>
+      <oui-dropdown-content>
+        <div role="menu" class="oui-pagination-menu">
+          <div class="oui-pagination-menu__group">Results per page</div>
+          <div class="oui-pagination-menu__items-list">
```

Comment:
`-list` -> What do you think about using `ul li` ?

Reply:
Already debated subject. We doesn't use div, as done in BS4.